src/modules/src/position_controller_pid.c

```diff
@@ -98,9 +100,9 @@ static struct this_s this = {
 
   .pidVZ = {
     .init = {
-      .kp = 25,
-      .ki = 15,
-      .kd = 0,
+      .kp = 3.0f,
```

Comment:
New tuning of the VZ PID. This needs to be tested in other positioning systems as well as it will effect those. I suggest testing the new values in Loco and Lighthouse systems

Reply:
Using Lighthouse: Very slowly descends on no input while using the assisted mode (less than a centimeter per second), and slightly overshoots on release of height control input (keeps going for about 5-10 cm). Have not been able to test it using the loco system